### PR TITLE
Add Laravel and PHPUnit Frameworks

### PR DESCRIPTION
**Pull request by josecanhelp**
Dependency management is done through composer, which is already covered.

# Laravel
Console: `php artisan tinker`
Server: `php artisan serve`

# PHPUnit
Test: `vendor/bin/phpunit`

### CODE REVIEW

File: plugins/3-laravel.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+laravel_console() {
+  run php artisan tinker
+}
+
+laravel_server() {
+  run php artisan serve
+}
+
+laravel_test() {
+  run vendor/bin/phpunit
```
Comment (bkeepers) [inline plugins/3-laravel.sh]: Is `phpunit` used outside of laravel? Is there a different config file that it looks for? In other words, is there ever not an `artisan` file but someone might still want `scrappy test` to run `phpunit`?

**Review by bkeepers — COMMENTED** [review]
Thanks for the pull request!

File: plugins/3-laravel.sh (hunk 1 above)
Comment (josecanhelp) [reply plugins/3-laravel.sh]: Yes, `phpunit` is a separate package from Laravel, but it's the default testing framework for Laravel. 

Would it be better to create a separate plugin file for `phpunit` that only runs the test command?

**Comment by bkeepers** [comment]
Awesome, thanks!

**Pull request merged**